### PR TITLE
6 article id comments

### PR DESCRIPTION
**Pull request by iwebsteraudio**

### CODE REVIEW

File: __tests__/int.test.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  })
+  test("Returns error 404 'Not Found' when given a valid request type which doesn't exists",()=>{
+    return request(app)
+    .get("/api/article/100000/comments")
+    .expect(404)
+    .then(({ body })=>{
+      const {msg} = body;
+      expect(msg).toBe("Not Found")
+    })
+  })
```
Comment (nc-richard-neat) [inline __tests__/int.test.js:191]: Nice testing here! The only thing we could also test for is what would happen if we grabbed an article that had no comments? This would be an empty array but isn't an error, and so we would need to think about how we can differentiate between this valid empty response and an empty response because the article doesn't exist. One to come back to at a later date 👍
